internal/dev_server/api/server.gen.go

```diff
@@ -1,6 +1,6 @@
 // Package api provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/oapi-codegen/oapi-codegen/v2 version v2.3.0 DO NOT EDIT.
+// Code generated by github.com/deepmap/oapi-codegen version v1.16.3 DO NOT EDIT.
```

Comment:
hmmmm how did you install the code generator? It looks like somehow you're using an old version.

Reply:
🤦 I thought I just did a `go install` but I had so many weird things going on locally.  Let me fix this.

Reply:
Oh the docs are kinda not great because they first tell you how to install the deprecated version. Try `go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@latest`